packages/shared/src/components/modals/UpvotedPopupModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ResponsiveModal } from './ResponsiveModal';
+import { ModalProps } from './StyledModal';
+import { Upvote } from '../../graphql/common';
+import { ModalCloseButton } from './ModalCloseButton';
+
+interface UpvotedPopupModalProps extends ModalProps {
+  upvotes?: Upvote[];
+  placeholderAmount?: number;
+}
+
+const containerClassName = 'flex flex-row px-6 mb-3';
+const contentClassName = 'flex flex-col flex-1 ml-4';
+const imageClassName = 'w-12 h-12 rounded-10 bg-pepper-70';
+
+const UpvotedPopupModal: React.FC<UpvotedPopupModalProps> = ({
+  upvotes,
+  placeholderAmount = 5,
+  onRequestClose,
+  ...modalProps
+}) => {
+  const renderPlaceholder = () =>
+    React.useMemo(
+      () =>
+        Array(placeholderAmount)
+          .fill(0)
+          .map(() => (
+            <div
+              key={Math.random()}
+              className={classNames(containerClassName, 'mt-3')}
+            >
+              <span className={imageClassName} />
+              <div className={contentClassName}>
+                <span className="rounded-10 bg-pepper-70 w-2/3 h-4" />
+                <span className="rounded-10 bg-pepper-70 w-1/3 h-4 mt-2" />
+              </div>
+            </div>
+          )),
+      [placeholderAmount],
+    );
+
+  const renderUpvoterList = () =>
+    React.useMemo(
+      () =>
+        upvotes?.map((upvote, i) => (
+          <div
```

Comment:
This element should link to the user's profile.
Please search for the `Link` component that we use from the nextjs framework.
There are lots of examples in the codebase.